src/pages/get-started.tsx

```diff
@@ -0,0 +1,287 @@
+import { FunctionComponent } from 'react'
+
+import classNames from 'classnames'
+import AppleIcon from 'mdi-react/AppleIcon'
+import CheckCircleIcon from 'mdi-react/CheckIcon'
+import ChevronRightIcon from 'mdi-react/ChevronRightIcon'
+import LaunchIcon from 'mdi-react/LaunchIcon'
+import MicrosoftWindowsIcon from 'mdi-react/MicrosoftWindowsIcon'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+import { Badge, ContentSection, Heading, HubSpotForm, Layout, Tabs } from '../components'
+
+const GetStartedPage: FunctionComponent = () => {
+    const router = useRouter()
+    const activeTab = (router.query.app as string) || 'install_app'
+
+    const handleTabChange = (active: string): void => {
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        router.push(`?app=${active}`)
+    }
+
+    const tabExtra = (
+        <div className="flex w-full flex-col gap-2 !text-left">
+            <p className="m-0 text-lg">You can also:</p>
+            <div className="flex gap-2 md:flex-col">
+                <Link
+                    target="_blank"
+                    href="https://github.com/sourcegraph/sourcegraph.git"
+                    className="flex items-center gap-1 text-lg"
+                >
+                    Build from source <LaunchIcon className="h-[12px] w-[12px] text-gray-700" />
+                </Link>
+                <Link target="_blank" href="https://sourcegraph.com" className="flex items-center gap-1 text-lg">
+                    Search public code <LaunchIcon className="h-[12px] w-[12px] text-gray-700" />
+                </Link>
+            </div>
+        </div>
+    )
+
+    return (
+        <Layout
+            meta={{
+                title: 'Install the Sourcegraph app',
+                description:
+                    'Install the app to get started with Sourcegraph quickly and easily. Use it to find, fix, and flow through all the code you care about, for free.',
+            }}
+            childrenClassName="bg-white"
+        >
+            <ContentSection>
+                <Tabs
+                    activeTabKey={activeTab || 'install_app'}
```

Comment:
If `activeTab` already has a default value assigned at the top of the component, let's avoid repetition here.
```suggestion
                    activeTabKey={activeTab}
```